server/src/collaborator/infra/entities/collaborator.entity.ts

```diff
@@ -1 +1,18 @@
-export class Collaborator {}
+import { randomUUID } from 'node:crypto';
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity('collaborator')
```

Comment:
Precisa de um deleted_at? Tem que analisar como vai funcionar essa parte do Login, tbm

Reply:
Não precisa, pois o colaborador que for desligado já perderá o acesso ao email corporativo